tests/test_utils/test_helpers.py

```diff
@@ -3,7 +3,10 @@
 import pytest
 import pandas as pd
 
-from src.utils.helpers import convert_formtype, values_in_column
+from src.utils.helpers import (
+    convert_formtype, values_in_column, tree_to_list
```

Comment:
haven't you removed the convert formtype function?

Reply:
No, it's still there. I just had to wrap a long line.

Reply:
I removed a duplicated block of code.